modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordLogsToolWindowFactory.kt

```diff
@@ -0,0 +1,21 @@
+package com.metalbear.mirrord
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.wm.ToolWindow
+import com.intellij.openapi.wm.ToolWindowFactory
+import com.intellij.ui.content.ContentFactory
+
+class MirrordLogsToolWindowFactory : ToolWindowFactory {
```

Comment:
Can the user disable this thing if they don't want to see it? And how do they enable it back?